site/content/docs/5.2/components/button-group.md

```diff
@@ -54,6 +61,15 @@ These classes can also be added to groups of links, as an alternative to the [`.
 
 Combine button-like checkbox and radio [toggle buttons]({{< docsref "/forms/checks-radios" >}}) into a seamless looking button group.
 
+<!-- Boosted mod : design callout -->
+{{< ods-incompatibility-alert >}}
+This checkbox variant can not be used because it does not respect the Orange Design System specifications.
+
+Instead use Boosted [Checks]({{< docsref "/forms/checks-radios" >}}) components.
```

Comment:
I don't agree with this suggestion because here the radio variant is not an equivalent of the checkbox in its functioning. The intent is to use the "real" checkbox from a functional and design point of view, and not checkbox that look more or less like buttons.

Reply:
Give more explanations of the why => new version of the message:
"This checkbox variant can not be used because it does not respect the Orange Design System specifications. From the design point of view, the checkbox should have the design of a checkbox as represented in our Boosted [Checks](https://...) component. 
For the correct design, you can also refer to the [Checkbox](https://system.design.orange.com/0c1af118d/p/88ab5b-forms/b/599459/i/48901789) component on on the Orange Design System website."

See if "see more/details summary" link to shorten the message would be need.